src/main/webapp/app/exercise/review/review-comment-widget-manager.ts

```
@@ -0,0 +1,292 @@
+        }
+        const widgetKey = this.getDraftKey(fileName, line);
+        this.editor.disposeWidgetsByPrefix(this.buildDraftWidgetId(fileName, line));
+        this.draftWidgetRefs.get(widgetKey)?.destroy();
+        this.draftWidgetRefs.delete(widgetKey);
+    }
+
+    /**
+     * Disposes all draft widget component refs.
+     */
+    private disposeDraftWidgets(): void {
+        this.draftWidgetRefs.forEach((ref) => ref.destroy());
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)